src/hotspot/os_cpu/bsd_aarch64/os_bsd_aarch64.cpp

```diff
@@ -523,14 +525,14 @@ extern "C" {
                                       // to entry for case SpinWait::NOP
         "  add  %[d], %[d], %[o]  \n"
         "  br   %[d]              \n"
-        "  b    SpinPause_return  \n" // case SpinWait::NONE  (-1)
+        "  b    1f  \n" // case SpinWait::NONE  (-1)
```

Comment:
If we're going forward with local labels, please try to align the newline characters with the rest